### PR TITLE
Add about us doc, add link, url and nav bar tab

### PR DESCRIPTION
**Pull request by nickyfantasy**
originated chinese doc from Wang Guan http://wiki.baidu.com/pages/viewpage.action?pageId=501931792

### CODE REVIEW

File: portal/portal/assets/markdown-docs/about_us_en.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# About us
+
+## What is PaddlePaddle
+ 
+ - PaddlePaddle is an open source deep learning framework provided by Baidu that enables developers and companies to implement their AI ideas safely and quickly.
+ - The team brings together the world's top deep learning scientists to provide developers and companies with the best deep learning experience
+ - The framework has four characteristics: easy to learn, easy to use, safe and efficient. It is the most suitable learning tool for Chinese developers and companies.
```
Comment (nickyfantasy) [inline portal/portal/assets/markdown-docs/about_us_en.md]: @varunarora Can you give us something to avoid mentioning Chinese?

Comment (varunarora) [reply portal/portal/assets/markdown-docs/about_us_en.md]: I can't. I think this entire doc is missing the point. I think these concerns should be brought up to someone in the senior management of the project.

**Comment by jetfuel** [comment]
@nickyfantasy I am not quite sure if its necessary to include the whole convert md to html, we can just serve the completed one directly.

**Comment by nickyfantasy** [comment]
@jetfuel ok, let me remove the script

File: portal/portal/urls.py [hunk 2]
```diff
@@ -57,6 +57,8 @@
     url(r'^docs/(?P<version>.*)/other/(?P<path>.*)$', views.other_path, name=url_helper.URL_NAME_OTHER),
     url(r'^docs/(?P<version>.*)/flush$', views.flush_other_page, name='flush_other_page'),
     url(r'^book$', views.book_home, name=url_helper.URL_NAME_BOOK_ROOT),
+    url(r'^about_en.html', views.about_en, name='about'),
```
[the remark this answers is not in the document]
Comment (nickyfantasy) [reply portal/portal/urls.py]: true

**Pull request merged**